test/mutation/mutation_pattern/test_target_aa_pattern.py

```diff
@@ -23,17 +23,16 @@ def test_decode_target_aa_pattern_not_expr():
     test_pattern = "all not self"
     test_orig_resi = "ARG"
     assert set(m_p.decode_target_aa_pattern(test_orig_resi, test_pattern)) == set([
-        "GLU", "ASN", "HIS", "SER", "VAL", "GLN", "TYR", "PRO", "ASP", "CYS", "GLY",
```

Comment:
Maybe an option prevent it from making shorter lines longer?

Reply:
See above. Lets figure it out in another PR